Mage.Server.Plugins/Mage.Deck.Constructed/src/mage/deck/CenturionCommander.java

```diff
@@ -66,21 +64,24 @@ public CenturionCommander() {
         banned.add("Tolarian Academy");
         banned.add("Treasure Cruise");
         banned.add("Vampiric Tutor");
-        banned.add("Vanishing");
-        banned.add("Winter Orb");
         banned.add("Yawgmoth's Bargain");
+        banned.add("Thassa's Oracle")
```

Comment:
missing `;`

Reply:
oh shit

Reply:
@JayDi85 thanks for your help, i not use this a lot